Orchid.Jint/JsCallback.cs

```diff
@@ -64,13 +91,13 @@ public object Apply(object @this, params object[] args)
             }
             catch (JavaScriptException jsError)
             {
-                Log.Warning("Scripting", "[{0}:{1}] {2}", jsError.Location.Source, jsError.LineNumber, jsError.Message);
+                Log.Warning("Scripting", $"[{_errorContext}{jsError.Location.Source}:{jsError.LineNumber}] {jsError.Message}");
```

Comment:
Perhaps consider adding some white spaces between each component to improve readability, but not necessary

Reply:
Leaving as is